org.kde.cervisia.json

```diff
@@ -0,0 +1,56 @@
+{
+  "app-id": "org.kde.cervisia",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "cervisia",
+  "rename-icon": "cervisia",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--talk-name=org.kde.klauncher5",
+    "--socket=system-bus"
```

Comment:
why system-bus? You should instead figure out what names are needed.

Reply:
Actually `session-bus` would have been correct, but I figured it out by looking at https://invent.kde.org/packaging/snapcraft-kde-applications/-/blob/Neon/release/cervisia/snapcraft.yaml#L32

Reply:
you can log the bus session when trying the build to see what is needed.